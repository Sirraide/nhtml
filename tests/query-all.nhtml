div
div
div

eval {
    for (const div of $$('div')) div.text = 'foo'
}