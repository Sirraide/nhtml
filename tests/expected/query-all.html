<div>foo</div>
<div>foo</div>
<div>foo</div>
